src/test/java/teammates/test/cases/action/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -268,16 +268,16 @@ public void testExecuteAndPostProcess() {
 
         assertEquals("", editCopyData.redirectUrl);
 
-        expectedString = "\"\" is not acceptable to TEAMMATES as a/an feedback session name because it is empty. "
+        expectedString = "The field feedback session name is empty. "
```

Comment:
On a related note, it is better if we can set the field name apart from the rest of the text. e.g. `The field 'feedback session name' is empty`. When it runs together with the rest of the test, it becomes hard to read. What do you think? Can be a separate PR.